### PR TITLE
Add statement about required license

### PR DESCRIPTION
**Pull request by dedemorton**
@lcawl @gchaps Can you help me get this language correct?

I've flagged the content as x-pack, but I'm already seeing users on the forum who don't understand why they can't see the option for central management.

Copying Gail because I plan to add a topic to the Kibana docs, too, and want to use similar language.

### CODE REVIEW

**Comment by lcawl** [comment]
You might consider advocating that they use the trial license to play with this feature. I have text like that in the first step here: https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-security.html (though don't use that old "X-Pack features" terminology).

**Comment by dedemorton** [comment]
@lcawl Great idea! I tweaked the text a bit to make it fit. Also decided to add it to another spot because people don't read carefully.

File: libbeat/docs/shared-license-statement.asciidoc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Don't have a license? You can start a 30-day trial. At the end of the trial
+period, you can purchase a subscription to keep using central management. For
+more information, see https://www.elastic.co/subscriptions and
+{xpack-ref}/license-management.html[License Management].
```
Comment (lcawl) [inline libbeat/docs/shared-license-statement.asciidoc]:
```suggestion
{stack-ov}/license-management.html[License Management].
```

File: libbeat/docs/shared-license-statement.asciidoc [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Don't have a license? You can start a 30-day trial. At the end of the trial
```
Comment (gchaps) [inline libbeat/docs/shared-license-statement.asciidoc]: @dedemorton In Kibana, you can start a 30-day trial in Management > License Management.  You might want to mention that in the topic that you add to the Kibana documentation.

**Pull request merged**